bin/helpers/capabilityHelper.js

```diff
@@ -286,6 +286,9 @@ const validate = (bsConfig, args) => {
     if (!Utils.isUndefined(bsConfig.run_settings.nodeVersion) && typeof(bsConfig.run_settings.nodeVersion) === 'string' && !bsConfig.run_settings.nodeVersion.match(/^(\d+\.)?(\d+\.)?(\*|\d+)$/))
         logger.warn(Constants.validationMessages.NODE_VERSION_PARSING_ERROR);
 
+    if(!Utils.isUndefined(cypressConfigFile.port)) {
```

Comment:
Can add a spec for this. Eg. https://github.com/browserstack/browserstack-cypress-cli/blob/007ae5150668fe0c00ae7cc477c3c70251e60452/test/unit/bin/helpers/capabilityHelper.js#L1200-L1207